azkaban-common/src/main/java/azkaban/executor/ExecutionFlowDao.java

```diff
@@ -462,10 +489,11 @@ public int selectAndUpdateExecutionWithLocking(final int executorId, final boole
   }
 
   /**
-   * This method is used to select executions in batch. It will apply lock and fetch executions.
-   * It will also update the status of those executions as mentioned in updatedStatus field.
-   * @param batchEnabled If set to true, fetch the executions in batch
-   * @param limit Limit in case of batch fetch
```

Comment:
it seems no change in this method,  just format change, please don't include it in the PR